feature/featuretags/featuretags.go

```diff
@@ -32,37 +34,89 @@ func (ft FeatureTag) OmitTag() string {
 	return "ts_omit_" + string(ft)
 }
 
+// DepSet returns the set of features that must be included to
+// use the given feature, including the feature itself.
+func DepSet(ft FeatureTag) set.Set[FeatureTag] {
+	s := set.Set[FeatureTag]{}
+	var add func(FeatureTag)
+	add = func(ft FeatureTag) {
+		if !ft.IsOmittable() {
+			return
+		}
+		s.Add(ft)
+		for _, dep := range Features[ft].Deps {
+			add(dep)
+		}
+	}
+	add(ft)
+	return s
+}
+
+// FeaturesWhichDependOn is the inverse of DepSet: it returns the set of features
+// that depend on the given feature, including the feature itself.
+func FeaturesWhichDependOn(ft FeatureTag) set.Set[FeatureTag] {
```

Comment:
Or maybe `RequiredBy`?

(Arguably we don't need "Features" in the name since this is already `featuretags`)